practice-one.css

```diff
@@ -0,0 +1,67 @@
+.nav{
+    align-items: flex-start;
+    display: flex;
+    height: 43px;
+    width: 1328px;
+}
```

Comment:
Missing empty line. You got this warning more 3 times, pls note. Next time, I will close your MR without notification